auto_round/autoround.py

```diff
@@ -920,11 +945,14 @@ def quant_block(self, block, input_ids, input_others, q_input=None, device=torch
         with torch.no_grad():
             unwrapper_block(block, best_v, best_min_scale, best_max_scale)
         if self.enable_quanted_input:
-
+            if self.block_wise:
+                block = block.to(device)
             q_outputs = self.get_block_outputs(
                 block, input_ids, input_others, self.train_bs * self.infer_bs_coeff, device,
                 cache_device=self.cache_device
             )
+            if self.block_wise:
+                block = block.to('meta')
```

Comment:
refine all the if and meta code, for example, provide a function, mv_module_from_gpu, move the module to meta if using block-wise, otherwise, moving to cpu